dts/arm/silabs/efm32wg.dtsi

```diff
@@ -70,6 +73,7 @@
 			reg = <0x4000e000 0x400>;
 			interrupts = <20 0>, <21 0>;
 			interrupt-names = "rx", "tx";
+			peripheral-id = <0>;
```

Comment:
`<3>`?

Reply:
@mnkp: thanks for the review! These peripheral-id values should be correct, since the intent of this patch is to map the correct peripheral (or module as it's called in the datasheet) to the correct `CMU_Clock_TypeDef` in em_cmu.h. The label of each peripheral is irrelevant in this case.
To clarify: in this case (efm32wg), there are 5 serial communication modules defined in the DT file, of which the first 3 are of type U**S**ART, are labelled 'UART1, UART2 and UART3' and should map to symbols `cmuClock_USART0`, `cmuClock_USART1` and `cmuClock_USART2`. 
The remaining 2 serial communication modules are of type UART, are labelleled 'UART3 and UART4' and should map to symbols `cmuClock_UART0` and `cmuClock_UART1`. Please let me know whether you agree or not.

Reply:
> The remaining 2 serial communication modules are of type UART, are labelleled 'UART3 and UART4' and should map to symbols cmuClock_UART0 and cmuClock_UART1. Please let me know whether you agree or not.

Definitely. That's the proper implementation. We care about the instance name as defined in the datasheet. I didn't expanded the collapsed code section and didn't catch this detail. My bad. Thanks for clarification.